lib/u3d/installer.rb

```diff
@@ -384,7 +397,70 @@ def exist?
     def editor_version
       require 'yaml'
       yaml = YAML.load(File.read("#{@path}/ProjectSettings/ProjectVersion.txt"))
-      yaml['m_EditorVersion']
+      version = yaml['m_EditorVersion']
+      if Helper.linux?
+        version.gsub!(/Linux/, '')
+        version.gsub!(/x/, '')
+      end
+      version
+    end
+  end
+
+  class LinuxDependencies
+    DEPENDENCIES = [
+      'gconf-service',
+      'lib32gcc1',
+      'lib32stdc++6',
+      'libasound2',
+      'libc6',
+      'libc6-i386',
+      'libcairo2',
+      'libcap2',
+      'libcups2',
+      'libdbus-1-3',
+      'libexpat1',
+      'libfontconfig1',
+      'libfreetype6',
+      'libgcc1',
+      'libgconf-2-4',
+      'libgdk-pixbuf2.0-0',
+      'libgl1-mesa-glx',
+      'libglib2.0-0',
+      'libglu1-mesa',
+      'libgtk2.0-0',
+      'libnspr4',
+      'libnss3',
+      'libpango1.0-0',
+      'libstdc++6',
+      'libx11-6',
+      'libxcomposite1',
+      'libxcursor1',
+      'libxdamage1',
+      'libxext6',
+      'libxfixes3',
+      'libxi6',
+      'libxrandr2',
+      'libxrender1',
+      'libxtst6',
+      'zlib1g',
+      'debconf',
+      'npm'
+    ]
+
+    def self.install_dependencies
+      if `which dpkg` != ''
+        prefix = 'apt-get -y install'
+      elsif `which rpm` != ''
+        prefix = 'yum -y install'
+      else
+        raise 'Cannot install dependencies on your Linux distribution'
+      end
+      DEPENDENCIES.each do |dep|
+        if UI.interactive?
+          next unless UI.confirm "Install #{dep}?"
+        end
+        U3dCore::CommandExecutor.execute(command: "#{prefix} #{dep}", admin: true)
```

Comment:
we can probably install all deps in a single command.
command: "#{prefix} #{DEPENDENCIES.join(' ')}"